### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by anderCM**
Let's finish our mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,38 +9,223 @@
...
+        <div class="technologies align-item">
+          <div class="technologie-label">
+            <p>html</p>
+          </div>
+          <div class="technologie-label">
+            <p>css</p>
+          </div>
+          <div class="technologie-label">
+            <p>javascript</p>
+          </div>
```
Comment (christianbiring1) [inline index.html]: - [x] Kindly make sure your tech stack list is grabbed inside a list of element.
`<ul>`
`<li>html</li>`
`<li>css</li>`
`</ul>`

File: index.html [hunk 2]
```diff
@@ -9,38 +9,223 @@
...
```
Comment (christianbiring1) [inline index.html]: - [x] Kindly make sure your tech stack list is grabbed inside a list of element.
`<ul>`
`<li>html</li>`
`<li>css</li>`
`</ul>`

File: index.html [hunk 3]
```diff
@@ -9,38 +9,223 @@
...
```
Comment (christianbiring1) [inline index.html]: - [x] Kindly make sure your tech stack list is grabbed inside a list of element.
`<ul>`
`<li>html</li>`
`<li>css</li>`
`</ul>`

File: index.html [hunk 4]
```diff
@@ -9,38 +9,223 @@
...
+        </div>
```
Comment (christianbiring1) [inline index.html]: - [x] Kindly make sure your tech stack list is grabbed inside a list of element.
`<ul>`
`<li>html</li>`
`<li>css</li>`
`</ul>`

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @anderCM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title`
- [ ] Great Ui :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [x] Kindly make sure your PR has a descriptive summary telling what changes were implemented in it.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @anderCM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions
- [x] [OPTIONAL]  If you add more informations about your changes  in this milestone to `PR` description, it would be better 😃 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (anderCM) [reply index.html]: Thanks for feedback, Tech skills will be wrapped in a `ul` and `li` tag

File: index.html (hunk 2 above)
Comment (anderCM) [reply index.html]: Thanks for feedback, Tech skills will be wrapped in a ul and li tag

File: index.html (hunk 3 above)
Comment (anderCM) [reply index.html]: Thanks for feedback, Tech skills will be wrapped in a ul and li tag

File: index.html (hunk 4 above)
Comment (anderCM) [reply index.html]: Thanks for feedback, Tech skills will be wrapped in a ul and li tag

**Pull request merged**